### ISSUE TITLE
[Feature Request] Cancel Pending Orders at n% {STOPLOSS}

### ISSUE THREAD

**Issue by Fogazi** [taniman/profit-trailer-enhancements#481]
A nice feature to enhance the use of strategies that use Pending Orders and current 'Stop Loss Trigger' would be to have a trigger to cancel a pending order if it reached n%(ie -10%) Then if you were a user of 'Stop Loss Trigger' also at -10% it would activate the stop loss and get rid of that pair.

Currently to do this you are required to manually cancel the pending order either in PT or on the exchange and then have PT's 'Stop Loss Trigger' activate at your set loss%

**Comment by Helmi** [comment]
Totally agree and woud have an additional request

DEFAULT_pending_cancel_above = -1
DEFAULT_pending_cancel_below = -10

This would also enable to cancel a pending order once it reached a certain profit in a positive way to maybe hand it over to sell strategies instead of just sell at a fixed price.

Big question for both cases: What happens to potentially existing other positions of the same coin?

**Comment by Fogazi** [comment]
Very good question and something I had failed to consider.

Perhaps then a whole new separate function to normal stop loss.

DEFAULT_pending_stop_loss = -10

This function would cancel the current pending order for the pair taking note of the amount of coins in the order and immediately market sell only the amount of the current pair that was in the pending order.

**Comment by Kriptork** [comment]
Yes, the cancel order would be great, Stoploss should override all other settings even if the coin is listed as pending.

**Comment by taniman** [comment]
@Kriptork  This issue as I mentioned above is for cancelling a pending order.  So that has already been implemented.

It is not possible to stoploss a pending order as that is an order that is on the exchange.  So the exchange takes over.  Only thing the bot does  with pending orders is cancel them

Seeing as I already implementing the cancel pending order I will close this now too.
